json_logic.gemspec

```diff
@@ -23,4 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler',  '~> 1.13'
   spec.add_development_dependency 'rake',     '~> 10.0'
   spec.add_development_dependency 'minitest', '~> 5.0'
+  spec.add_development_dependency 'pry'
```

Comment:
Love pry. Thanks for adding.

Reply:
I actually forgot to take it out, but yeah - it's how I found it without spending a bunch of time. ;)